docs/getting-started/welcome_mhairi.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+slug: /
+---
+
+UMA stands for "Universal Market Access" 
+
+<!--taken from UMA/TRS Mission and Values -->
+Our mission is to make financial markets universally fair and accessible.
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
+UMA builds open source infrastructure which enables developers to create financial instruments in a permissionless, secure and efficient manner.  We work with developers, financial engineers and product designers to support them build products on our infrastructure, while supporting tokenholders to actively participate in the governance of our protocol.
```

Comment:
Recommend something like this: 

 UMA is open source infrastructure for developers to easily create permissionless, secure and efficient financial instruments.

Reply:
Then dive into what the infrastructure is and what it's for (synthetic assets to start)